### PR TITLE
Clarify docstrings on TimeSeries

### PR DESCRIPTION
**Pull request by eteq**
Prompted by the scipy2015 Bokeh tutorial.  The docstring on `values` for `TimeSeries` is currently quite opaque and difficult to understand.  This change makes things a bit more verbose for the Bokeh novice who just does ``TimeSeries?`` in an IPython notebook.

### CODE REVIEW

**Comment by damianavila** [comment]
Probably the docstrings is not the better in this case, but the "could be anything" does not sounds proper to me... @eteq can you rephrase it in another way? @fpliger, thoughts?

**Comment by fpliger** [comment]
Agree. It's definitely and improvement but " anything that can be converted" is quite broad without defining a conversion method. Explicit is better then implicit here.

That said `charts` is going through a big refactoring and this is going to change anyway.. the plan is to be much more accurate about what we support and what not. I'd say I'm +0 to merge it as is and use this better spelling as base for a more specific docstring in the next version.

**Comment by damianavila** [comment]
OK, let's merge this one now to be improved later.
Thanks @eteq for your contribution.

**Comment by eteq** [comment]
Oops, sorry, for some reason I missed the notification of this so that's why I didn't rephrase it better when you asked, @damianavila and @fpliger
